Issues with Source-Code Build Documentation

So, this is the first time I ever tried to do this Code Build with Gradle and get the Jar thing, and the Readme seems to be a little bit wrong for me. And I had to figure out what worked for me, so I gonna be write this down.

I have `Windows 8.1 x64 Professional` Operation System, and:

* I successfully installed `JDK 7 (v79) x64` and `Java 7 (v79) x64` and added the Variables correctly.
* I successfully installed `Gradle v1.11` and added the Variables correctly. (Because I tried to solve things, I followed to download this version instead of v2.6) 
* I successfully installed `Git 2.5.0 x64`.

Then I came to this: **Setup EE3**

The first thing that was not the same in my case is I didn't got an `mcdev` folder. I didn't deal with this issue.

And the second, very important thing is if I download EE3's source into the `C:\Github\Equivalent-Exchange-3\` directory, the actual files will be in this directory: `C:\Github\Equivalent-Exchange-3\Equivalent-Exchange-3\`, because the download creates another folder. And hardly I figured out that's why the `gradle setupCiWorkspace` command didn't want to work, because the instructions didn't say to navigate to `C:\Github\Equivalent-Exchange-3\Equivalent-Exchange-3\`, and everytime I tried to execute the command, it didn't work, because I was navigated to the `C:\Github\Equivalent-Exchange-3\` folder (the steps before this said to navigate to here) and there was nothing here except another `Equivalent-Exchange-3` folder.

I already solved the problem, just wanted to note that I think at least the second thing is not documented correctly, and I hope you understand what I try to say. If you want the gradle error that was before I figured out the problem, here it is:
```
C:\Windows\system32>cd C:\GitHub\Equivalent-Exchange-3\

C:\GitHub\Equivalent-Exchange-3>gradle setupCiWorkspace

FAILURE: Build failed with an exception.

* What went wrong:
Task 'setupCiWorkspace' not found in root project 'Equivalent-Exchange-3'.

* Try:
Run gradle tasks to get a list of available tasks. Run with --stacktrace option
to get the stack trace. Run with                                               -
-info
or                                                                             -
-debug
option to get more log output.

BUILD FAILED

Total time: 1.425 secs
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,0 +1,28 @@
+## Contributing
+First of all, thanks for contributing! :sparkles: :+1:
+
+### Submitting a PR
+So you found a :bug: in the code?  Think you can make it more efficient :dash:?  Want to help in general?  Great!
+
+1. If you haven't already, create a Github account.
+2. Click the `Fork` icon located at the top-right of this page (below your username).
+3. Make the changes that you want to and commit them.
+	* If you're making changes locally, you'll have to execute `git commit -a` and `git push` in your command line.
+4. Click `Pull Request` at the right-hand side of the gray bar directly below your fork's name.
+5. Click `Click to create a pull request for this comparison`, enter your PR's title, and create a detailed description telling pahimar what you changed.
+6. Click `Send pull request`, and wait for feedback! :boom:
+
+### Creating an Issue
+EE3 crashes every time :bomb:?  Have a suggestion?  Found a :bug:?  Create an issue now!
+
+1. Make sure your issue hasn't already been answered or fixed by [searching for it](https://github.com/pahimar/Equivalent-Exchange-3/search?q=&type=Issues).  Also think about whether your issue is a valid one before submitting it.
+	* Please do not open an issue to ask a question-that is for [pahimar's Twitter](https://twitter.com/pahimar/) or the [forums](http://www.minecraftforum.net/topic/1540010-equivalent-exchange-3).
+2. Go to [the issues page](http://github.com/pahimar/Equivalent-Exchange-3/issues).
+3. Click `New Issue` right below `Star` and `Fork`.
+4. Enter your Issue's title (something that summarizes your issue), and then create a detailed description ("Hey pahimar, could you add/change xxx?" or "Hey, found an exploit:  stuff").
+	* If you are reporting a bug report from an unofficial version, make sure you include the following:
+		* Commit SHA (usually located in a changelog or the jar name itself)
+		* ForgeModLoader log
+		* Server log if applicable
+		* Detailed description of the bug and pictures if applicable
+5. Click `Submit new issue`, and wait for feedback! :boom:

--- a/README.md
+++ b/README.md
@@ -100,7 +100,7 @@ In order to get the most up-to-date builds, you'll have to periodically update y
 1. Open up your command line.
 2. Navigate to wherever you cloned EE3 in the console.
 3. Make sure you have not made any changes to the local repository, or else there might be issues with Git.
-	* If there are local changes, try reverting them to the status that they were when you last updated your repository by executing `git revert HEAD --hard`.
+	* If there are local changes, try reverting them to the status that they were when you last updated your repository by executing `git reset HEAD --hard`.
 4. Execute `git pull master`.  This pulls all commits from the official repository that do not yet exist on your local repository and updates it.
 
 ### Contributing

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
-##Welcome to Equivalent Exchange 3!
-**LATEST OFFICIAL VERSION**: [All versions available here](http://equivalent.exchange)
+## Welcome to Equivalent Exchange 3!
+[All versions are available here](http://equivalent.exchange)
 
 [Minecraft Forums page](http://www.minecraftforum.net/topic/1540010-equivalent-exchange-3)
 
@@ -9,9 +9,9 @@
 
 [FAQ](https://github.com/pahimar/Equivalent-Exchange-3/wiki/Frequently-Asked-Questions) - For those that have questions.
 
-###Compiling Equivalent Exchange 3
-IMPORTANT: Please report any issues you have, there might be some problems with the documentation!
-Also make sure you know EXACTLY what you're doing!  It's not any of our faults if your OS crashes, becomes corrupted, etc.
+### Compiling Equivalent Exchange 3
+IMPORTANT: Please report any issues you have as there might be some problems with the documentation.
+Also make sure you know EXACTLY what you're doing before proceeding!  We are not responsible if your computer crashes, becomes corrupted, etc. :see_no_evil:
 ***
 [Setup Java](#setup-java)
 
@@ -25,7 +25,7 @@ Also make sure you know EXACTLY what you're doing!  It's not any of our faults i
 
 [Updating Your Repository](#updating-your-repository)
 
-####Setup Java
+#### Setup Java
 The Java JDK is used to compile EE3.
 
 1. Download and install the Java JDK.
@@ -45,7 +45,7 @@ The Java JDK is used to compile EE3.
     * Append `;%JAVA_HOME%\bin` EXACTLY AS SHOWN and click `Ok`.  Make sure the location is correct; double-check just to make sure.
 3. Open up your command line and run `javac`.  If it spews out a bunch of possible options and the usage, then you're good to go.  If not, either try the steps again or check the [FAQ](https://github.com/pahimar/Equivalent-Exchange-3/wiki/Frequently-Asked-Questions).
 
-####Setup Gradle
+#### Setup Gradle
 Gradle is used to execute the various build tasks when compiling EE3.
 
 1. Download and install Gradle.
@@ -63,49 +63,50 @@ Gradle is used to execute the various build tasks when compiling EE3.
 	* Scroll down to `Path` again, and append `;%GRADLE_HOME%\bin` EXACTLY AS SHOWN and click `Ok`.  Once again, double-check the location.
 3. Open up your command line and run `gradle`.  If it says "Welcome to Gradle [version].", then you're good to go.  If not, either try the steps again or check the [FAQ](https://github.com/pahimar/Equivalent-Exchange-3/wiki/Frequently-Asked-Questions).
 
-####Setup Git
+#### Setup Git
 Git is used to clone EE3 and update your local copy.
 
 1. Download and install Git [here](http://git-scm.com/download/).
 	* *Optional*: Download and install a Git GUI client, such as Github for Windows/Mac, SmartGitHg, TortoiseGit, etc.  A nice list is available [here](http://git-scm.com/downloads/guis).
 
-####Setup EE3
+#### Setup EE3
 This section assumes that you're using the command-line version of Git.
 
 1. Open up your command line.
-2. Navigate to a place where you want to download EE3's source (eg `C:\Github\Equivalent-Exchange-3\`) by executing `cd [folder location]`.  This location is known as `mcdev` from now on.
-3. Execute `git clone https://github.com/pahimar/Equivalent-Exchange-3.git`.  This will download EE3's source into `mcdev`.
+2. Navigate to a place where you want to download EE3's source (eg `C:\Github`) by executing `cd [folder location]`.  If choosing a location other than `C:\Github`, just remember that when following the instructions.
+3. Execute `git clone https://github.com/pahimar/Equivalent-Exchange-3.git`.  This will download EE3's source into an `Equivalent-Exchange-3` folder
+such as `C:\Github\Equivalent-Exchange-3`.
 4. Right now, you should have a directory that looks something like:
 
 ***
-	mcdev
+	Github
 	\-Equivalent-Exchange-3
 		\-EE3's files (should have `build.gradle`)
 ***
 
-####Compile EE3
-1. Execute `gradle setupCiWorkspace`. This sets up Forge and downloads the necessary libraries to build EE3.  This might take some time, be patient.
-	* You will generally only have to do this once until the Forge version in `build.properties` changes.
+#### Compile EE3
+1. Execute `gradle setupCiWorkspace` in order to set up Forge and download the necessary libraries to build EE3.  This might take some time, please be patient :watch:.
+	* You will generally only have to do this when the Forge version in `build.properties` changes.
 2. Execute `gradle build`. If you did everything right, `BUILD SUCCESSFUL` will be displayed after it finishes.  This should be relatively quick.
     * If you see `BUILD FAILED`, check the error output (it should be right around `BUILD FAILED`), fix everything (if possible), and try again.
-3. Navigate to `mcdev\Equivalent-Exchange-3\build\libs`.
-    *  You should see a `.jar` file named `EquivalentExchange3-1.6.4-0.1.#.jar`, where # is the `build_number` value in `build.properties`.
-		* NOTE: `null` means that you are missing a `build_number` value in `build.properties` or that your CI environment is set up incorrectly.
+3. Navigate to `C:\Github\Equivalent-Exchange-3\build\libs`.
+    *  You should see a `.jar` file named `EquivalentExchange3-x.y.z-#.jar`, where x.y.z is the Minecraft version number and # is the mod version.
+		* NOTE: `null` means that you are missing a `build_number` value in `build.properties` or that your CI environment is set up incorrectly-it is totally safe to ignore.
 4. Copy the jar into your Minecraft mods folder, and you are done!
 
-####Updating Your Repository
+#### Updating Your Repository
 In order to get the most up-to-date builds, you'll have to periodically update your local repository.
 
 1. Open up your command line.
-2. Navigate to `mcdev` in the console.
+2. Navigate to wherever you cloned EE3 in the console.
 3. Make sure you have not made any changes to the local repository, or else there might be issues with Git.
-	* If you have, try reverting them to the status that they were when you last updated your repository.
+	* If there are local changes, try reverting them to the status that they were when you last updated your repository by executing `git revert HEAD --hard`.
 4. Execute `git pull master`.  This pulls all commits from the official repository that do not yet exist on your local repository and updates it.
 
-###Contributing
+### Contributing
 ***
-####Submitting a PR
-So you found a bug in pahimar's code?  Think you can make it more efficient?  Want to help in general?  Great!
+#### Submitting a PR
+So you found a :bug: in the code?  Think you can make it more efficient :dash:?  Want to help in general?  Great!
 
 1. If you haven't already, create a Github account.
 2. Click the `Fork` icon located at the top-right of this page (below your username).
@@ -113,19 +114,19 @@ So you found a bug in pahimar's code?  Think you can make it more efficient?  Wa
 	* If you're making changes locally, you'll have to execute `git commit -a` and `git push` in your command line.
 4. Click `Pull Request` at the right-hand side of the gray bar directly below your fork's name.
 5. Click `Click to create a pull request for this comparison`, enter your PR's title, and create a detailed description telling pahimar what you changed.
-6. Click `Send pull request`, and wait for feedback!
+6. Click `Send pull request`, and wait for feedback! :boom:
 
-####Creating an Issue
-EE3 crashes every time?  Have a suggestion?  Found a bug?  Create an issue now!
+#### Creating an Issue
+EE3 crashes every time :bomb:?  Have a suggestion?  Found a :bug:?  Create an issue now!
 
-1. Make sure your issue hasn't already been answered or fixed.  Also think about whether your issue is a valid one before submitting it.
+1. Make sure your issue hasn't already been answered or fixed by [searching for it](https://github.com/pahimar/Equivalent-Exchange-3/search?q=&type=Issues).  Also think about whether your issue is a valid one before submitting it.
 	* Please do not open an issue to ask a question-that is for [pahimar's Twitter](https://twitter.com/pahimar/) or the [forums](http://www.minecraftforum.net/topic/1540010-equivalent-exchange-3).
 2. Go to [the issues page](http://github.com/pahimar/Equivalent-Exchange-3/issues).
 3. Click `New Issue` right below `Star` and `Fork`.
-4. Enter your Issue's title (something that summarizes your issue), and then create a detailed description ("Hey pahimar, could you add/change xxx?" or "Hey, found an exploit:  stuff").
+4. Enter your Issue's title (something that summarizes your issue), and then create a detailed description ("Hey pahimar, could you add/change xxx?" or "Hey, found an exploit: stuff").
 	* If you are reporting a bug report from an unofficial version, make sure you include the following:
 		* Commit SHA (usually located in a changelog or the jar name itself)
 		* ForgeModLoader log
 		* Server log if applicable
 		* Detailed description of the bug and pictures if applicable
-5. Click `Submit new issue`, and wait for feedback!
+5. Click `Submit new issue`, and wait for feedback! :boom:

